draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -5932,6 +5934,11 @@ Expect: 100-continue
    field in the response unless it can verify that the request is a duplicate
    of an immediately prior change made by the same user agent.
 </t>
+<t>
+   If the origin server does not have a last modification date available for
+   the selected representation, the If-Modified-Since header field &MUST;
+   be ignored.
+</t>
```

Comment:
That's a new normative requirement, right? I do not object, but it would need to be called out in changes-from-rfc723*.

Reply:
Perhaps it would be easier to state that the condition evaluates to false if the target resource has no selected representation or if the selected representation has no last modification date available, since both would indicate that the resource has changed.

Reply:
It's already implied by the new clause above, I think it's OK to just delete this new requirement.